utils/dataloaders.py

```diff
@@ -1085,3 +1088,48 @@ def _hub_ops(f, max_dim=1920):
     if verbose:
         print(json.dumps(stats, indent=2, sort_keys=False))
     return stats
+
+
+class ClassificationDataset(torchvision.datasets.ImageFolder):
+    """
+    YOLOv5 Classification Dataset.
+    Arguments
+        root:  Path of the dataset
+        transform:  torchvision transforms, used as default transforms
+        album_transform: Albumentations transform, used if installed
+    """
+
+    def __init__(self, root, transform, album_transform=None):
```

Comment:
@glenn-jocher should we rename `transforms` to `torch_transforms`?

Reply:
Yeah sure!

Reply:
But wait keep them both plural or singular, i.e. _transform or _transforms

Reply:
Okk.